python/pyspark/ml/clustering.py

```diff
@@ -1209,9 +1209,9 @@ class PowerIterationClustering(HasMaxIter, HasWeightCol, JavaParams, JavaMLReada
     .. note:: Experimental
 
     Power Iteration Clustering (PIC), a scalable graph clustering algorithm developed by
-    `Lin and Cohen <http://www.icml2010.org/papers/387.pdf>`_. From the abstract:
+    `Lin and Cohen <http://www.cs.cmu.edu/~frank/papers/icml2010-pic-final.pdf>`_. From the
     PIC finds a very low-dimensional embedding of a dataset using truncated power
-    iteration on a normalized pair-wise similarity matrix of the data.
+    abstract: iteration on a normalized pair-wise similarity matrix of the data.
```

Comment:
Could you check this again? It seems to break the original sentence accidentally. Maybe, `From the abstract:`?